src/main/java/org/mskcc/cmo/messaging/impl/JSGatewayImpl.java

```diff
@@ -150,15 +150,14 @@ public void subscribe(String subject, Class messageClass,
             ConsumerConfiguration consumerConfig = ConsumerConfiguration.builder()
                     .durable(consumerName)
                     .filterSubject(filterSubject)
-                    .deliverPolicy(DeliverPolicy.New)
                     .ackPolicy(AckPolicy.All)
-                    .replayPolicy(ReplayPolicy.Instant)
+                    .deliverPolicy(DeliverPolicy.New)
                     .build();
             PushSubscribeOptions options = PushSubscribeOptions.builder()
                     .configuration(consumerConfig)
                     .build();
             JetStreamSubscription sub = jsConnection.subscribe(subject, dispatcher,
-                msg -> onMessage(msg, messageClass, messageConsumer), true, options);
+                msg -> onMessage(msg, messageClass, messageConsumer), false, options);
```

Comment:
why the change from true to false?

Reply:
The boolean here is autoack and this should be set to false because we have an ackPolicy setup with the consumers.